src/com/ivoyant/java/threads/ThreadsExample.java

```diff
@@ -0,0 +1,39 @@
+package com.ivoyant.java.threads;
+
+class A extends Thread{
```

Comment:
Instead of A,B,C... take some use cases and also make sure u give a proper class names everytime